website/docs/admin_hosts_maya.md

```diff
@@ -107,6 +107,18 @@ or Deadlines **Draft Tile Assembler**.
 This is useful to fix some specific renderer glitches and advanced hacking of Maya Scene files. `Patch name` is label for patch for easier orientation.
 `Patch regex` is regex used to find line in file, after `Patch line` string is inserted. Note that you need to add line ending.
 
+## Include/exclude handles by task type
+You can include or exclude handles globaly or by task type.
```

Comment:
```suggestion
You can include or exclude handles globally or by task type.

This setting is used to define the default frame range for new scenes or when using the **OpenPype > Set frame range** setting.
```

This made me think - does this actually also influence the **Validate Frame Range** validations or whatever it's called where it validates during publish whether an instance matches the asset document's frame range + handles?

Reply:
Yes it does influence the Validate Frame Range validator. And the "Repair" button when the validator fails repair correctly the framerange and the handles in the failed instances.